section18/Auto.rb

```diff
@@ -0,0 +1,25 @@
+class Auto
+  def initialize()
+		@placa = "#{("a".."z").to_a.sample} - #{rand(10..99)} #{("a".."z").to_a.sample}"
```

Comment:
@sansuaza The indentation needs to be fixed. By code styling rule: 1 tab -> translated into 2 spaces